src/App.js

```diff
@@ -79,6 +80,7 @@ class App extends Component {
             />
             <Route path="/login/" exact component={Login} />
             <Route path="/signup/" exact component={Signup} />
+            <Route path="/forgotpassword/" exact component={ForgotPassword} />
```

Comment:
What do you think about

```suggestion
            <Route path="/forgot-password/" exact component={ForgotPassword} />
```

Reply:
changed